src/cljs/pyregence/components/map_controls/camera_tool.cljs

```diff
@@ -187,21 +173,12 @@
                                   (when-let [new-camera (js->clj (aget features "properties") :keywordize-keys true)]
                                     (u-async/stop-refresh! @exit-chan)
                                     (reset! active-camera new-camera)
-                                    (reset! camera-age 0)
                                     (reset! image-src nil)
                                     (let [image-chan (get-camera-image-chan @active-camera)]
-                                      (reset! camera-age (as-> (:update-time @active-camera) %
-                                                               ;; Alert Wildfire and Alert California have differently formatted `:update-time` values
-                                                               (if (= (:api-name @active-camera) "alert-wildfire")
-                                                                 (u-time/alert-wf-camera-time->js-date %)
-                                                                 (js/Date. %))
-                                                               (u-time/get-time-difference %)
-                                                               (u-time/ms->hr %)))
-                                      (when (> 4 @camera-age)
-                                        (reset! image-src (<! image-chan))
-                                        (reset! exit-chan
-                                                (u-async/refresh-on-interval! #(go (reset! image-src (<! (get-camera-image-chan @active-camera))))
-                                                                              60000)))))))
+                                      (reset! image-src (<! image-chan))
```

Comment:
A second pair of eyes on this change would be nice as I haven't reviewed it thoroughly.